src/interfaces/NodegenRc.ts

```diff
@@ -1,13 +1,13 @@
 export interface Helpers {
-  publishOpIds?: string[],
-  subscribeOpIds?: string[],
+  publishOpIds?: string[];
+  subscribeOpIds?: string[];
 
-  [helperName: string]: any
+  [helperName: string]: any;
 }
 
-export interface InjectionOverwrites {
-  targetDir: string,
-  url: string
+export interface Injection {
+  destination?: string;
```

Comment:
this is redundant at the moment - should be removed, hints at recipes but this is not what this feature does